website/data/toc.yaml

```diff
@@ -103,6 +103,8 @@ sections:
         url: /docs/contributors/testing
       - name: Code Organization
         url: /docs/contributors/codebase
+      - name: Deploying to Maven Central
```

Comment:
I question whether we need to publish this as a site doc. Maybe this could just be a markdown file under release/, since it's really only for our internal audience for now. We could move it to the site later.